skrf/tests/test_vectorfitting.py

```diff
@@ -101,9 +105,9 @@ def test_read_write_npz(self):
         self.assertTrue(np.allclose(vf.proportional_coeff, vf2.proportional_coeff))
         self.assertTrue(np.allclose(vf.constant_coeff, vf2.constant_coeff))
 
+    @pytest.mark.skipif("matplotlib" in sys.modules, reason="Raise Error only if matplotlib is not installed.")
```

Comment:
The test is usually not skipped because matplotlib is no more in the dependencies, is it like that ?

Reply:
This test is skipped for the default test runs (py37 to py311) as matplotlib is installed and no exception is raised.
The test is executed in the minimal setup and asserts the RuntimeError

Reply:
Great!